content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -226,15 +226,9 @@ CSI volume expansion requires enabling `ExpandCSIVolumes` feature gate and also
 
 You can only resize volumes containing a file system if the file system is XFS, Ext3, or Ext4.
 
-When a volume contains a file system, the file system is only resized when a new Pod is started using
-the `PersistentVolumeClaim` in ReadWrite mode. Therefore, if a pod or deployment is using a volume and
-you want to expand it, you need to delete or recreate the pod after the volume has been expanded by the cloud provider in the controller-manager. You can check the status of resize operation by running the `kubectl describe pvc` command:
-
-```
-kubectl describe pvc <pvc_name>
-```
-
-If the `PersistentVolumeClaim` has the status `FileSystemResizePending`, it is safe to recreate the pod using the PersistentVolumeClaim.
+When a volume contains a file system, the file system is only resized when a new Pod is using
+the `PersistentVolumeClaim` in ReadWrite mode. File System expansion is either done when Pod is starting up
```

Comment:
Here's two suggestions; I hope they're useful:
```suggestion
the `PersistentVolumeClaim` in ReadWrite mode. File system expansion is either done when Pod is starting up
```

<br />

or possibly:
```suggestion
the PersistentVolumeClaim in `ReadWrite` mode. File system expansion is either done when a Pod is starting up
```
(closer to style guide)